haas/cli.py

```diff
@@ -166,25 +172,58 @@ def user_create(username, password, is_admin):
 
 @cmd
 def network_create(network, creator, access, net_id):
-    """Create a link-layer <network>.  See docs/networks.md for details"""
-    url = object_url('network', network)
-    do_put(url, data={'creator': creator,
-                      'access': access,
-                      'net_id': net_id})
+    """Create a link-layer <network>.  See docs/networks.md for details
+            --          --              --              --
+        network: Any string to name the network
+        creator: Either Project name, or  string "admin"
+         access : Either Project name, or  null (eg. "")
+         net-id : Either a vlan-id, or let HaaS assign one (eg. "")
+
+         Eg: haas network_create prod-network01 admin "" "" [or]
+             haas network_create prod-network02 admin proj-01 ""
+    """
+    try: 
+        ret = h.network_create(network, creator, access, net_id)
+        print "    SUCCESS: Network %s created " % network
+    except DuplicateName as e:
+        print e.message
+    except IncompleteDependency as e:
+        print e.message
+    except UnknownError as e:
+        print e.message
+
+
 
 @cmd
 def network_create_simple(network, project):
     """Create <network> owned by project.  Specific case of network_create"""
-    url = object_url('network', network)
-    do_put(url, data={'creator': project,
-                      'access': project,
-                      'net_id': ""})
+
+    try:
+        ret = h.network_create_simple(network, project) 
+        print "    SUCCESS: Network %s created " % network
```

Comment:
In thinking about this some more, maybe we should not print anything on success, and print the error on failure?